source/services/encryption/EncryptionHelper.ts

```diff
@@ -0,0 +1,68 @@
+import StorageService from '../StorageService';
+
+export interface UserInterface {
+  personalNumber: string;
+}
+
+export interface FormsInterface {
+  answers: { encryptedAnswers: string };
+  encryption: {
+    type: string;
+    publicKey: {
+      P: number;
+      G: number;
+      symmetricKeyName: string;
+      publicKeys: Record<number, undefined | string>;
+    };
+  };
+}
+
+export function EncryptionException(message: string) {
+  this.message = message;
+  this.name = 'EncryptionException';
+}
+
+export function getPublicKeyInForm(personalNumber: string, forms: FormsInterface) {
+  return forms.encryption.publicKey.publicKeys[personalNumber];
+}
+
+function getSymmetricKeyStorageKeyword(forms: FormsInterface) {
+  return `${forms.encryption.publicKey.symmetricKeyName}`;
+}
+
+export async function getStoredSymmetricKey(forms: FormsInterface) {
+  const storageKeyword = getSymmetricKeyStorageKeyword(forms);
+  const symmetricKey = await StorageService.getData(storageKeyword);
+  return Number(symmetricKey);
+}
+
+export async function storeSymmetricKey(symmetricKey: number, forms: FormsInterface) {
+  const storageKeyword = getSymmetricKeyStorageKeyword(forms);
+  await StorageService.saveData(storageKeyword, symmetricKey.toString());
+}
+
+// Prof of concept, will be replaced with Diffie-Hellman library.
+export function getPseudoKey(G: number, privateKey: number, P: number) {
+  if (privateKey === 1) {
+    return G;
+  }
+
+  return G ** privateKey % P;
+}
```

Comment:
Suggest we add a task in clickup telling that we need to implement the DH lib instead. Easy to forget by just relying on the comment. :)

Reply:
Skapat ärende https://app.clickup.com/t/nz7wwt i clickup.